projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -110,6 +110,26 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
     if (this.resetMenuOnClose) {
       this.resetOnMenuCollapse();
     }
+    if (
+      this.featureConfigService?.isEnabled('a11yNavigationUiKeyboardControls')
+    ) {
+      this.focusOnMenuExpansion();
+    }
+  }
+
+  /**
+   * Focus on the first focusable element in the hamburger menu when the menu is opened.
+   */
+  focusOnMenuExpansion(): void {
+    this.subscriptions.add(
+      this.hamburgerMenuService?.isExpanded.pipe().subscribe((isExpanded) => {
+        if (isExpanded && this.navAriaLabel?.includes('menu')) {
+          setTimeout(() => {
+            this.elemRef.nativeElement.querySelector('[tabindex="0"]').focus();
+          }, 0);
```

Comment:
Just a question, no need to change anything. Do we still need to pass `0` to `setTimeout`? Does it work fine without it?

Reply:
Should work fine without and since it defaults to zero, I've removed it. ;)